conduit/data/datasets/vision/celeba.py

```diff
@@ -10,7 +10,8 @@
 import torch
 
 from conduit.data.datasets.utils import FileInfo, ImageTform, download_from_gdrive
-from conduit.data.datasets.vision.base import CdtVisionDataset
+
+from .base import CdtVisionDataset
```

Comment:
Why make this change? The absolute path is perfectly functional and is what auto-import uses.

Reply:
it's clearer that you're importing from a file that's in the same module

Reply:
Local imports are less likely to be cyclic

Reply:
But such imports are also more brittle in the face of refactoring